index.html

```diff
@@ -1655,6 +1655,33 @@ <h2>Contexts and Vocabularies</h2>
 Verifiable Credentials v2.0: Vocabularies</a>.
         </p>
 
+        <section>
+          <h3>Context Injection</h3>
+
+          <p>
+The `@context` property is used to ensure that implementations are using the
+same semantics when terms in this specification are processed. For example, this
+can be important when properties like `type` are processed and its value, such
+as `DataIntegrityProof`, are used.
+          </p>
+
+          <p>
+If a `@context` property is not provided in a document that is being secured or
+verified, or the Data Integrity terms used in the document are not defined by
+existing values in the `@context` property, implementations MUST inject or add
+an `@context` property with a value of
+`https://www.w3.org/ns/data-integrity/v1`.
+          </p>
+
+          <p>
+An example of when context injection is unnecessary is when the Verifiable
+Credential Data Model v2.0 context (`https://www.w3.org/ns/credentials/v2`)
+exists as a value in the `@context` property, as that context defines all of the
+necessary Data Integrity terms defined by
+`https://www.w3.org/ns/data-integrity/v1`.
```

Comment:
Just flagging for ourselves: before going to CR we will have to crosscheck the (1) credential vocabulary (2) credential context (3) data integrity context and (4) data integrity vocabulary that all these statements are indeed correct. These vocabularies change almost constantly these days, and it is easy to miss out something...